Cargo.toml

```diff
@@ -10,10 +10,27 @@ include = ["**/*.rs", "Cargo.toml", "README.md", ".gitignore"]
 
 [dependencies]
 blake2 = "0.10.6"
-rand_core = "0.6.4"
 
 [dev-dependencies]
+rand = "0.8.5"
+rand_core = "0.6.4"
 rand_chacha = "0.3.1"
+criterion = { version = "0.5.1", features = ["html_reports"]}
+
+[[bench]]
+name = "centralized_time"
+harness = false
+path = "benches/centralized_telescope.rs"
+
+[[bench]]
+name = "centralized_step"
+harness = false
+path = "benches/centralized_telescope.rs"
```

Comment:
We used to have separate files but @curiecrypt asked to merge them in one (I guess in prevision of the next protocols).
Funnily, we cannot have the proof and criterion bench in the same file because of the criterion main macro